requirements.txt

```
@@ -10,49 +10,49 @@ itsdangerous==0.24 # pyup: ignore
 git+https://github.com/alphagov/digitalmarketplace-utils.git@48.5.0#egg=digitalmarketplace-utils==48.5.0
 git+https://github.com/alphagov/digitalmarketplace-content-loader.git@5.2.1#egg=digitalmarketplace-content-loader==5.2.1
 git+https://github.com/alphagov/digitalmarketplace-apiclient.git@19.15.1#egg=digitalmarketplace-apiclient==19.15.1
-git+https://github.com/alphagov/govuk-frontend-jinja.git@v0.4.0-alpha#egg=govuk-frontend-jinja==0.4.0-alpha
+git+https://github.com/alphagov/govuk-frontend-jinja.git@v0.4.0-alpha#egg=govuk-frontend-jinja==0.5.0-alpha
 
 ## The following requirements were added by pip freeze:
-asn1crypto==0.24.0
+asn1crypto==1.2.0
 blinker==1.4
-boto3==1.9.214
-botocore==1.12.214
-certifi==2019.6.16
-cffi==1.12.3
+boto3==1.10.7
+botocore==1.13.7
+certifi==2019.9.11
+cffi==1.13.1
 chardet==3.0.4
 Click==7.0
-contextlib2==0.5.5
+contextlib2==0.6.0.post1
 cryptography==2.3.1
 defusedxml==0.6.0
 docopt==0.6.2
 docutils==0.15.2
 Flask-Script==2.0.6
 fleep==1.0.1
-future==0.17.1
+future==0.18.2
 gds-metrics==0.2.0
 govuk-country-register==0.5.0
 idna==2.8
 inflection==0.3.1
-Jinja2==2.10.1
+Jinja2==2.10.3
 jmespath==0.9.4
 mailchimp3==3.0.6
 Markdown==2.6.11
 MarkupSafe==1.1.1
 monotonic==1.5
-notifications-python-client==5.3.0
+notifications-python-client==5.4.0
 odfpy==1.4.0
 prometheus-client==0.2.0
 pycparser==2.19
 PyJWT==1.7.1
 python-dateutil==2.8.0
 python-json-logger==0.1.11
-pytz==2019.2
+pytz==2019.3
 PyYAML==3.13
 requests==2.22.0
 s3transfer==0.2.1
 six==1.12.0
 unicodecsv==0.14.1
-urllib3==1.25.3
-Werkzeug==0.15.5
+urllib3==1.25.6
+Werkzeug==0.16.0
```

We need to keep Werkzeug < 0.16.0
```suggestion
Werkzeug==0.15.5
```